README.md

```diff
@@ -55,7 +55,7 @@ $ pytest test
 
 ## API changes
 
-If you make changes to the spec (and thus the API) of the StaticNarrative app, you will need to recompile the app to generate updated versions of `StaticNarrativeImpl.py` and `StaticNarrativeServer.py`.
+If you make changes to the spec (and thus the API) of the StaticNarrative app, you will need to recompile the app to generate updated versions of `StaticNarrativeImpl.py`, `StaticNarrativeServer.py`, and the compilation report, `compile_report.json`. These updated files must be committed to the GitHub repo.
```

Comment:
Maybe this should include a pull request checklist to ensure the developers do these steps!!

Reply:
Good idea, thanks!